_troubleshooting/timescaledb/background-worker-failed-start.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Failed to start a background worker
+section: troubleshooting
+topics: [jobs, continuous aggregates, data retention, compression]
+errors:
+  - language: bash
+    message: |-
+      "<TYPE_OF_BACKGROUND_JOB>": failed to start a background worker
+apis:
+  - [jobs, timescaledb_information.jobs]
+  - [jobs, timescaledb_information.job_stats]
+  - [jobs, timescaledb_information.add_job()]
+  - [hypertables, add_reorder_policy()]
+  - [hypertables, add_retention_policy()]
+  - [hypertables, add_compression_policy()]
+  - [hypertables, add_continuous_aggregate_policy()]
+keywords: [jobs, policies, user-defined actions]
+tags: [jobs, scheduled jobs, background jobs, background workers, automation framework, policies, user-defined actions]
+---
+
+<!---
+* Keep this section in alphabetical order
+* Use this format for writing troubleshooting sections:
+ - Cause: What causes the problem?
+ - Consequence: What does the user see when they hit this problem?
+ - Fix/Workaround: What can the user do to fix or work around the problem? Provide a "Resolving" Procedure if required.
+ - Result: When the user applies the fix, what is the result when the same action is applied?
+* Copy this comment at the top of every troubleshooting page
+-->
+
```

Comment:
Missing title?

Reply:
This is for the reorganized troubleshooting section! (I created a bit of a maintenance headache for myself by being too quick to reorganize the troubleshooting entries. After having to update them a few times to keep them in sync, I decided the easiest way was to merge the thing and make a parallel edit every time we have a PR that edits the current troubleshooting pages.)

In the proposed new troubleshooting format, you don't need the title and error message in the body! They'll be pulled from frontmatter.

The second file is the page that will be changed on our current live site and it has both in the message body :)

Reply:
Right! OK 👍